tslint.json

```diff
@@ -63,6 +63,7 @@
     "no-outputs-metadata-property": true,
     "template-banana-in-box": true,
     "use-lifecycle-interface": true,
-    "linebreak-style": [true, "LF"]
+    "linebreak-style": [true, "LF"],
+    "no-magic-numbers": [true, 0, 1, -1]
```

Comment:
as tslint "no-magic-numbers" rule is so poor (not configurable, and default config is stupid), lets remove that rule since we dont have eslint

Reply:
reverted the rule